src/test/java/com/library/steps/US03_StepDef.java

```diff
@@ -0,0 +1,54 @@
+package com.library.steps;
+
+import com.library.pages.BookPage;
+import com.library.pages.LoginPage;
+import com.library.utility.BrowserUtil;
+import com.library.utility.DB_Util;
+import io.cucumber.java.en.*;
+import org.junit.Assert;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class US03_StepDef {
+
+    LoginPage loginPage = new LoginPage();
+    BookPage bookPage = new BookPage();
+    List<String> actualBookCategories = new ArrayList<>();
+    @Given("the user logged in as {string}")
+    public void the_user_logged_in_as(String userType) {
+       loginPage.login(userType);
+    }
+    @When("the user navigates to {string} page")
+    public void the_user_navigates_to_page(String module) {
+        bookPage.navigateModule(module);
+    }
+    @When("the user gets all book categories in webpage")
+    public void the_user_gets_all_book_categories_in_webpage() {
+        actualBookCategories = BrowserUtil.getAllSelectOptions(bookPage.categoryDropdown);
+        actualBookCategories.remove(0);
+        System.out.println("actualBookCategories = " + actualBookCategories);
```

Comment:
Hiiiii, Everything looks Perfect, Just one thing for us to practice that in real work we should delete " System.out.println" to save the running time. if i'm not wrong.

thank you :)